core/src/main/kotlin/pages/ContentNodes.kt

```diff
@@ -182,11 +245,17 @@ data class ContentDivergentGroup(
 ) : ContentComposite {
     data class GroupID(val name: String)
 
-    override val sourceSets: Set<DokkaSourceSet>
+    override val sourceSets: Set<ContentSourceSet>
         get() = children.flatMap { it.sourceSets }.distinct().toSet()
 
     override fun withNewExtras(newExtras: PropertyContainer<ContentNode>): ContentDivergentGroup =
         copy(extra = newExtras)
+
+    override fun transformChildren(transformer: (ContentNode) -> ContentNode): ContentDivergentGroup =
+        copy(children = children.map(transformer).filterIsInstance<ContentDivergentInstance>())
+
+    // TODO NOW?
```

Comment:
There is no intuitive behaviour for me here. I think this should be tackled when we think about ContentNode<->SourceSet associations in the future!